addons/rbac/allow-kubeadm-join-configmap.yaml

```diff
@@ -21,6 +21,7 @@ rules:
 - apiGroups:
   - ""
   resourceNames:
+  - kubelet-config
```

Comment:
Any reason why the 1.24 configmap doesn't have a version suffix? Is it generic for all 1.24+ releases?

Reply:
https://github.com/kubermatic/kubermatic/pull/9736#discussion_r867901759

Reply:
they changed it for 1.24 to drop the suffix.